src/UI/UserFactory.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.Identity.UI
+{
+    internal class UserFactory<TUser, TKey> : IUserFactory<TUser>
```

Comment:
Nit: what do you think about making this public instead of internal?  Not a big deal either way but I could see this being somewhat useful potentially.

Reply:
Yeah, should be ok, but will leave it as it is for now as we are going to revisit this api for rc1